examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -22,6 +32,7 @@ const user = generateUser() as any;
 const frsAzUser = {
     userId: user.id,
     userName: user.name,
+    additionalDetails: userDetails,
```

Comment:
Let's also add the code to consume and render these `additionalDetails` in `renderAudience` in view.ts to show the full loop, from providing the user details for the token provider to seeing them be returned in the audience